rest/shared/src/main/scala/io/udash/rest/DefaultRESTConnector.scala

```diff
@@ -12,14 +12,33 @@ import monix.reactive.Observable
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.implicitConversions
 
+sealed trait Protocol
+object Protocol {
+  case object Http extends Protocol
+  case object Https extends Protocol
+
+  def apply(s: String): Option[Protocol] = s match {
+    case "http:" | "http" => Some(Http)
+    case "https:" | "https" => Some(Https)
+    case _ => None
+  }
+}
+
 /** Default implementation of [[io.udash.rest.internal.RESTConnector]] for Udash REST. */
-class DefaultRESTConnector(val host: String, val port: Int, val pathPrefix: String)(implicit val ec: ExecutionContext) extends RESTConnector {
+class DefaultRESTConnector(protocol: Protocol, val host: String, val port: Int, val pathPrefix: String)(implicit val ec: ExecutionContext) extends RESTConnector {
```

Comment:
Add `val` to `protocol`, it should be consistent with `host`, `port` and `pathPrefix`.